data/db_demo.yaml

```diff
@@ -131,6 +157,24 @@ allocation:
     hours_allocated: 100
     group_id: =program_A
     instrument_id: =SEDM
+  - pi: Michael Coughlin
+    proposal_id: LT-001
+    start_date: "3020-02-12T00:00:00"
+    end_date: "3020-07-12T00:00:00"
+    hours_allocated: 100
+    group_id: =program_A
+    instrument_id: =IOO
+    username: fritz_bot
+    password: pmB5G5BeQM
```

Comment:
@mcoughlin needless to say, but this password is now compromised (ie it's out in the open). If it was real, then you should change it immediately even before figuring out how to store it secretly.

Reply:
@mcoughlin If you push a new commit getting rid of this password, it will still exist in the repo via the commit history. If you can't get a new password from the LT folks, can you prune it out using bfg-repo cleaner? 

https://rtyley.github.io/bfg-repo-cleaner/